packages/core/src/components/graphs/area.ts

```diff
@@ -2,12 +2,16 @@
 import { Component } from "../component";
 import * as Configuration from "../../configuration";
 import { CartesianOrientations, Events } from "../../interfaces";
+import { GradientUtils } from "../../services";
+import { Tools } from "../../tools";
 
 // D3 Imports
 import { area } from "d3-shape";
+import { select } from "d3";
```

Comment:
importing something from `d3` rather than the specific d3 package will cause all of `d3` to be imported and there will be no tree-shaking available at that point.

I believe this should say `import { select } from "d3-selection";`